akeneo-design-system/src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,7 +1,21 @@
-import React from 'react';
+import React, {Ref, useState} from 'react';
 import styled, {css, keyframes} from 'styled-components';
 import {AkeneoThemedProps, getColor} from 'theme';
 import {CheckIcon, PartialCheckIcon} from 'icons';
+import {useShortcut} from 'hooks/use-shortcut';
```

Comment:
```suggestion
import {useShortcut} from 'hooks';
```
And add an `index.ts` in the hooks folder